tests/pytests/unit/modules/test_mac_brew_pkg.py

```diff
@@ -115,6 +115,52 @@ def custom_call_brew(*cmd, failhard=True):
                       "tap": "custom/tap",
                       "url": "https://iterm2.com/downloads/stable/iTerm2-3_4_3.zip",
                       "version": "3.4.3"
+                    },
+                    {
+                      "token": "discord",
+                      "full_token": "discord",
+                      "tap": null,
```

Comment:
This is the current behavior for the `tap` value when you ask `brew` for the installed packages.